### PR TITLE
feat: llamaindex and langchain

### PR DESCRIPTION
**Pull request by santiagxf**
# Description

This PR introduces examples for LangChain and LlamaIndex integrations with Azure AI Foundry.

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure-Samples/azureai-samples/blob/main/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure-Samples/azureai-samples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

File: scenarios/langchain/tracing-with-langchain.ipynb [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.projects import AIProjectClient\n",
+    "from azure.identity import DefaultAzureCredential\n",
+    "\n",
+    "project_client = AIProjectClient.from_connection_string(\n",
+    "    credential=DefaultAzureCredential(),\n",
+    "    conn_str=\"<region>.api.azureml.ms;<tenant>;<hub>;<project>\",\n",
```
Comment (qubitron) [inline scenarios/langchain/tracing-with-langchain.ipynb:105]: We shouldn't document the format of the connection string, it implies that they need to construct it manually. Suggest to:
 - change this to <project_connection_string>
 - tell them where to get the connection string from (the project overview page)

File: scenarios/langchain/getting-started-with-langchain-embeddings.ipynb [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from langchain_azure_ai.embeddings import AzureAIEmbeddingsModel\n",
+    "\n",
+    "embed_model = AzureAIEmbeddingsModel(\n",
+    "    endpoint=os.environ[\"AZURE_INFERENCE_ENDPOINT\"],\n",
```
Comment (qubitron) [inline scenarios/langchain/getting-started-with-langchain-embeddings.ipynb:64]: should we have a version of this that just takes a project connection string and model name? that way users don't need to store the inference endpoint/credential in plain text

**Review by qubitron — APPROVED** [review]
Left a couple of comments, please fix the project connection string before merging!

File: scenarios/langchain/tracing-with-langchain.ipynb (hunk 1 above)
Comment (santiagxf) [reply scenarios/langchain/tracing-with-langchain.ipynb]: Done!

File: scenarios/langchain/getting-started-with-langchain-embeddings.ipynb (hunk 2 above)
Comment (santiagxf) [reply scenarios/langchain/getting-started-with-langchain-embeddings.ipynb:66]: Let's do this next release!

**Pull request merged**